src/redux/books/books.js

```diff
@@ -35,20 +18,38 @@ const addremoveReducer = (state = data, action = {}) => {
     case REMOVE_BOOK: {
       return [...state.filter((book) => book.id !== action.payload)];
     }
+    case GET_BOOKS: {
+      return action.payload;
+    }
     default:
       return state;
   }
 };
 
 // Action Creators
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
-
-export const removeBook = (payload) => ({
-  type: REMOVE_BOOK,
-  payload,
-});
+
+export const fetchBooks = () => async (dispatch) => {
```

Comment:
- [ ] According to the project requirements, we are required to use `createAsyncThunk` for creating our thunk action creators. Correct me if I'm wrong, I do not see that you have used it
![Screenshot from 2022-11-10 18-03-54](https://user-images.githubusercontent.com/46388113/201172733-a67336c5-6778-441b-80fa-9cf2d3985ec9.png)

Reply:
I think the project requirement is to either use `fetch` or `axios` with `createAsyncThunk`.
so I am not using axios and not using createAsyncThunk.